src/unstakeAg/types.ts

```diff
@@ -216,3 +217,7 @@ export interface ExchangeReturn {
   unstakeTransaction: TransactionWithSigners;
   cleanupTransaction?: TransactionWithSigners;
 }
+
+export interface ExchangeReturnV0 {
+  unstakeTransaction: VersionedTransaction;
+}
```

Comment:
dont we still need `Signers[]` for creating the new stake account in the case of lido? 

P.S. we can remove this once we move to our stakedex onchain program since all the signers will be PDAs

Reply:
what should be the `Signers[]`?

Reply:
in the case of unstakexsol https://github.com/igneous-labs/unstake-ag/blob/341bdd806b17f1c79f2e9ac472ad0767b788488e/src/unstakeAg/index.ts#L769 if `mustUseKeypairForSplitStake()` is true then the new stake account keypair is an additional signer